### PR TITLE
feat: add sample code for create and delete document schema

### PR DESCRIPTION
**Pull request by paulinawins**
## Description

Note: due to workflow auth issues, this PR is non-forked unlike last PR for delete document schema.

## Checklist
- [x] I have followed guidelines from [CONTRIBUTING.MD](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md) and [Samples Style Guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [x] **Tests** pass:   `npm test` (see [Testing](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#run-the-tests-for-a-single-sample))
- [x] **Lint** pass:   `npm run lint` (see [Style](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#style))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] This pull request is from a branch created directly off of `GoogleCloudPlatform/nodejs-docs-samples`. Not a fork.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new sample directory, and I created [GitHub Actions workflow](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#adding-new-samples) for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Review by holtskinner — APPROVED** [review]

File: document-warehouse/test/delete-document-schema.test.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+const {describe, it} = require('mocha');
+const {assert} = require('chai');
+const sinon = require('sinon');
+
+const text = 'Document Schema Deleted';
+
+const {deleteDocumentSchema} = require('../delete-document-schema');
+
+// TODO(developer): Fill in the variables below before running this test.
+const projectId = 'YOUR_PROJECT_ID';
```
Comment (pattishin) [inline document-warehouse/test/delete-document-schema.test.js:26]: If you are planning on using a real project - do you mind leveraging the `DocumentSchemaServiceClient` to infer the project w/ `getProjectId()`. 
This should be picked up with the current GCP project we have this hooked up to.

Here's an example from another sample: https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/iam/deny/test/deny.test.js#L32

Ref: https://github.com/GoogleCloudPlatform/nodejs-docs-samples/actions/runs/5859762543/job/15886428603?pr=3423#step:11:39

**Review by pattishin — DISMISSED** [review]
It looks like this will require you to have a real schema to be deleting from the GCP project. Can you add accompanying resources to ensure the resource exists before deleting?

File: document-warehouse/test/delete-document-schema.test.js (hunk 1 above)
Comment (paulinawins) [reply document-warehouse/test/delete-document-schema.test.js]: Included projectId pulling for tests. Note that I added create document schema for test completeness as well.

**Comment by pattishin** [comment]
Hi @paulinawins! Do you mind checking out that `document-warehouse` test failure?

**Review by pattishin — APPROVED** [review]

**Pull request merged**